auto/src/glew_init_gl.c

```diff
@@ -5,8 +5,22 @@ GLboolean GLEWAPIENTRY glewGetExtension (const char* name)
   const GLubyte* start;
   const GLubyte* end;
   start = (const GLubyte*)glGetString(GL_EXTENSIONS);
```

Comment:
Is glGetString(GL_EXTENSIONS) even legal on a core context?  Should we prefer glGetIntegerv(GL_NUM_EXTENSIONS), and fall back to the compatible method?